Data Structures/Stacks/stack_paren_balanced.py

```diff
@@ -0,0 +1,24 @@
+s=input()
+l=[]
+def is_match(a,b):
+  if a=="(" and b==")":
+    return True
+  if a=="{" and b=="}":
+    return True 
+  if a=="[" and b=="]":
+    return True
+flag=True
+for i in range(0,len(s)):
+  if s[i] in "({[" and flag==True:
+    l.append(s[i])
+  else:
+    if len(l)==0:
```

Comment:
@Sachindrck Kindly use proper spaces between `==` and other such statements as per the [Google's Python Style Guide](http://google.github.io/styleguide/pyguide.html).